### PR TITLE
Governance and Donate page redesign

### PR DESCRIPTION
**Pull request by garbas**
# Goverance page

## Design

- [Governance_desktop.pdf]( https://github.com/NixOS/nixos-artwork/blob/master/www/2020_redesign/Governance/Governance_deskop.pdf)
- [Governance_mobile.pdf]( https://github.com/NixOS/nixos-artwork/blob/master/www/2020_redesign/Governance/Governance_mobile.pdf)

## TODO

- [x] "Learn more" link in RFC process section should be green button (@garbas)
- [x] new image for rfc process, recreate in inkscape (@samueldr)
- [x]  ‼️ dark blue _diagonal_ background for Teams section (@samueldr)
- [x] icons for teams (need export from the design @garbas)
- [x] design team listing, font and green button (@garbas)
- [x] add "NixOS foundation" logo missing in mission statement (@samueldr)
- [x] design listing of board members as in design (@garbas)
- [x] design Team pages (@garbas)

# Donate page

## Design
- [Donate_deskop.pdf](https://github.com/NixOS/nixos-artwork/blob/master/www/2020_redesign/Donate/Donate_deskop.pdf)
- [Donate_mobile.pdf](https://github.com/NixOS/nixos-artwork/blob/master/www/2020_redesign/Donate/Donate_mobile.pdf)

## TODO

- [x] :bangbang: vertical "NixOS Foundation" logo, needs some inkscape work (@samueldr)
- [x] _diagonal_ light blue background for stores section (@samueldr)
- [ ] image of some fictional character wearing NixOS t-shirt (designer via @garbas)
- [x] add both stores to the store section (@garbas)
- [x] prepare text for open collective (@garbas)
- [x] 3 columns for sponsors in Acknowledgements section, also size of images should be the same (@garbas)

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixos-homepage/588)

### CODE REVIEW

**Comment by samueldr** [comment]
I'm left with the "Foundation" logo to handle.

Though I see you've made pretty good use of text replacement to use the existing logo+logotype, but with a text-based "Foundation" word.

What is desired for the "Foundation" word? I like the idea of having a thinner "Foundation" word under the NixOS word, like currently implented:

image

Maybe [*Vegur ExtraLight*](https://www.fontsquirrel.com/fonts/vegur) rather than the currently inherited font.

Any other thoughts?

**Comment by garbas** [comment]
> Though I see you've made pretty good use of text replacement to use the existing logo+logotype, but with a text-based "Foundation" word.
> 

Yeah I just want to move forward, but in case you don't have time I just implemented a CSS based version that looks similar.

> What is desired for the "Foundation" word? I like the idea of having a thinner "Foundation" word under the NixOS word, like currently implented:
> 
> image
> 
> Maybe [_Vegur ExtraLight_](https://www.fontsquirrel.com/fonts/vegur) rather than the currently inherited font.
> 
> Any other thoughts?

Yes, use the same font as the NixOS text but lighter. `Vegur ExtraLight` sounds about right.

**Comment by samueldr** [comment]
:bangbang: The foundation logo has been implemented in #589

**Comment by garbas** [comment]
I merge #589 and master ... we have only few things left to finish this PR.

**Comment by samueldr** [comment]
> > > icons for teams (need export from the design @garbas)
> > 
> > 
> > Are the icons from the PDF not okay? If they are I can try extracting them, chances are it'll work just fine since it's not a complicated bit in the PDF.
> 
> Yes the icons are good from the PDF just I don't know how to extract them.

Good, I'll handle *exporting* them at the very least.

The gist of it is importing the PDF in inkscape, it will generally handle things fine. Though sometimes it requires a bit of cleanup for things like reducing the amount of points by "simplifying" the paths, and such, so the files end up smaller. Ideally, one would also try to "pixel-snap" lines so they don't end up fuzzy if shown at 1:1 scale.

**Comment by samueldr** [comment]
The graphical elements have been extracted, only. They are "face12"'d so they can be coloured with svg.less. If you need more help for them just ask.

They're all exported with 138x138px bounding boxes, so you can apply them to element "centered and contained" and they will be the right scale comparative to each other.

**Review by davidak — CHANGES_REQUESTED** [review]
overall it looks very good. well done!

File: donate.tt [hunk 1]
```diff
@@ -4,119 +4,165 @@
...
-    <div>
-      <a href="https://www.hetzner.de/"><img src="[% root %]logo/hetzner-online.svg" alt="Hetzner Online GmbH" /></a>
+          <li>
+            <a href="https://www.macstadium.com/">
+              <img src="[% root %]logo/macstadium.png" alt="MacStadium logo" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.fastly.com/">
+              <img src="[% root %]logo/fastly.png" alt="Fastly" />
```
[the remark this answers is not in the document]
Comment (garbas) [reply donate.tt:139]: @davidak how did you get to the glitch?

[the remark this answers is not in the document]
Comment (davidak) [reply donate.tt:139]: open in browser? i mean that the white space around the fastly is not a rectangle like the other, but the corners are cut out, like you see in the screenshot

**Pull request merged**